cart-page.css

```diff
@@ -0,0 +1,83 @@
+.cart-container {
+  display: grid;
+  grid-template-columns: 65% 35%;
+  height: 100%;
+  width: 87%;
+  margin: auto;
+  margin-top: 90px;
+  margin-bottom: 90px;
+}
+
+.cart-items {
+  display: flex;
+  flex-direction: column;
+}
+
+.sm-btn {
+  padding: 5px 10px;
+  height: fit-content;
+  font-size: 1.1rem;
+}
+
+.card-btn-row {
+  align-items: center;
+}
+
+.flex-row.sm {
+  gap: 7px;
+}
+
+.qty-row input {
+  width: 30px;
+}
+.qty-row button:hover {
+  cursor: pointer;
+}
+
+.qty-row button {
+  color: var(--dark-grey);
+  background-color: var(--yellow-color-light);
+  width: 20px;
+  border: none;
+  font-size: 0.8rem;
+}
+
+.buy-container {
+  border-left: 2px solid var(--light-grey);
+  margin-left: 30px;
+  width: 100%;
+  height: 100%;
+}
+.divider {
+  margin: auto;
+  width: 90%;
+  background-color: var(--light-grey);
+  height: 1px;
+}
+.n-wt {
+  font-weight: 400;
+}
+.img-container {
+  object-fit: cover;
+}
+
+.cart-details {
+  padding: 20px;
```

Comment:
use rem

Reply:
I dont want it to scale but might consider when i work on the responsiveness of the site